vocode/streaming/streaming_conversation.py

```diff
@@ -461,6 +461,12 @@ def broadcast_interrupt(self):
                         num_interrupts += 1
             except queue.Empty:
                 break
+        # TODO(julien/ajay) Currently, the message cut off logic won't run if we cancel
+        # We should probably add it for consistency
+        # self.synthesis_results_worker.cancel_current_task()
```

Comment:
i think we actually don't want to cancel the current task for the synthesis results worker - we want `send_speech_to_output` to close gracefully and return `message_sent`

Reply:
I think you're right but it might be good codebase hygiene to handle the exception to establish a consistent pattern. I don't feel strongly about it - I was just leaving it here so it's on the radar.